packages/@sanity/cli/package.json

```
@@ -62,6 +62,8 @@
   },
   "dependencies": {
     "@babel/traverse": "^7.19.0",
+    "@vercel/frameworks": "^1.3.2",
```

As `@vercel/fs-dectors` already depend on `@vercel/frameworks` you should be able to drop it :smil'
```suggestion
    "@vercel/frameworks": "^1.3.2",
```